lib/contracts/contributor.js

```diff
@@ -8,9 +9,13 @@ class Contributor extends Record {
 
   getById(id) {
     return this.functions.getContributorById(id)
-      .then((data) => {
+      .then(data => {
         return this.ipfs.catAndMerge(data, ContributorSerializer.deserialize);
-      });
+      })
+      .then(data => {
+        data.balanceInt = KreditsFormatter(data.balance);
+        return Promise.resolve(data);
```

Comment:
No need for `Promise.resolve` because every `then` returns a new Promise.

`return data;` should work.